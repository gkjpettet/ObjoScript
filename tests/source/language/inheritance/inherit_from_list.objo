// Expect runtime error: 
// Class `Subclass` cannot inherit from built-in class `List`.
class Subclass is List {}