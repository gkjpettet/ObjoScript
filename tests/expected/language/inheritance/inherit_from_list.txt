Class `Subclass` cannot inherit from built-in class `List`.